interfaces/xenmgr_vm.xml

```diff
@@ -139,6 +139,9 @@
     <property name="argo" type="b" access="readwrite">
       <tp:docstring>Use Argo.</tp:docstring>
     </property>
+    <property name="authenforce" type="b" access="readwrite">
+      <tp:docstring>Enable or disable Authentication enforcement mode in UEFI guest VMs.</tp:docstring>
```

Comment:
Is authenforce false the equivalent of setup mode?  What are the implications of "Enable or disable Authentication enforcement"?